.editorconfig

```diff
@@ -182,6 +189,16 @@ csharp_indent_case_contents_when_block = true
 csharp_indent_switch_labels = true
 csharp_indent_labels = flush_left
 
+# Whitespace options
+csharp_style_allow_embedded_statements_on_same_line_experimental = false:warning
+dotnet_diagnostic.IDE2001.severity = warning
+
+csharp_style_allow_blank_lines_between_consecutive_braces_experimental = false:warning
+dotnet_diagnostic.IDE2002.severity = warning
+
+csharp_style_allow_blank_line_after_colon_in_constructor_initializer_experimental = false:warning
+dotnet_diagnostic.IDE2004.severity = warning
```

Comment:
❗ All of the warning diagnostics need to move under the IDE section (starting line 257 in this file) unless the compiler separately signed off on enabling them.